sycl/test/abi/sycl_symbols_linux.dump

```diff
@@ -3524,14 +3528,14 @@ _ZN4sycl3_V17handler20associateWithHandlerEPNS0_6detail30UnsampledImageAccessorB
 _ZN4sycl3_V17handler20memcpyToDeviceGlobalEPKvS3_bmm
 _ZN4sycl3_V17handler20setKernelCacheConfigENS1_23StableKernelCacheConfigE
 _ZN4sycl3_V17handler20setStateSpecConstSetEv
+_ZN4sycl3_V17handler21setKernelWorkGroupMemEm
```

Comment:
Since this symbol hasn't changed but simply moved in the ordering, I've been advised previously to remove such changes from PRs (or submit them as a separate PR) to aid in reviewing these changes.